homeassistant/components/sensor/imap.py

```diff
@@ -60,15 +63,16 @@
 class ImapSensor(Entity):
     """Representation of an IMAP sensor."""
 
-    def __init__(self, name, user, password, server, port, folder):
+    def __init__(self, name, user, password, server, port, folder, search):
         """Initialize the sensor."""
         self._name = name or user
         self._user = user
         self._password = password
         self._server = server
         self._port = port
         self._folder = folder
-        self._unread_count = 0
+        self._email_count = 0
```

Comment:
I know you didn't really change this but we should probably set the state to `None` while the actual value is unknown.

Reply:
Changed!